src/snackbar.jsx

```diff
@@ -162,7 +161,7 @@ const Snackbar = React.createClass({
 
       //Only Bind clickaway after transition finishes
       this.timerTransitionId = setTimeout(() => {
-        this._bindClickAway();
+        this.timerTransitionId = 0;
```

Comment:
@newoga This isn't necessary really.

> Passing an invalid ID to clearTimeout does not have any effect (and doesn't throw an exception).

From https://developer.mozilla.org/en-US/docs/Web/API/WindowTimers/clearTimeout